tests/engine/cwd-option.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { depngn } from '../../src';
+
+describe('cwd option', () => {
+	it('possible to perform the check in the existing directory passing a relative path', async () => {
+		const checkResult = await depngn({ version: '18.0.0', cwd: '.' });
+		expect(checkResult).not.toBeNull();
```

Comment:
@KostiantynPopovych Can we actually expect something a bit more specific? This is the same expectation as specified in the next test case.

Reply:
@etagwerker, thank you for reviewing! And it is a great point. I'll take a look at what we can expect here)

Reply:
@KostiantynPopovych sounds good. Did you get a chance to improve this part?

Reply:
@etagwerker just pushed my update, thank you.